lib/chef/resource/timezone.rb

```diff
@@ -112,6 +118,14 @@ def current_darwin_tz
             /Time Zone: (.*)/.match(tz_shellout.stdout)[1]
           end
         end
+        def current_windows_tz
+          tz_shellout = shell_out!("tzutil /g")
+          if /is not recognized as an internal/.match?(tz_shellout.stderr)
```

Comment:
If you use the shell_out without the !
```
tz_shellout = shell_out("tzutil /g")
```
Then you can test the exitstatus rather than raising directly
```
if tz_shellout.exitstatus == 1
```
This will save you matching the stderr from the command processor.

Reply:
thanks for the advice, I've amended it, is that OK?

Reply:
@stuartpreston hi there, I added a test to cover off windows.  Was that what you had in mind?

Thanks for your help